src/runtime-rs/crates/resource/src/share_fs/share_virtio_fs.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::{fs, path::Path};
+
+use anyhow::{Context, Result};
+use hypervisor::{device, Hypervisor};
+use kata_sys_util::mount;
+
+use super::utils::{get_host_ro_shared_path, get_host_rw_shared_path};
+
+pub(crate) const MOUNT_GUEST_TAG: &str = "kataShared";
+pub(crate) const PASSTHROUGH_FS_DIR: &str = "passthrough";
+
+pub(crate) const TYPE_VIRTIO_FS: &str = "virtio_fs";
```

Comment:
Recommend `TYPE_VIRTIO_FS` -> `FS_TYPE_VIRTIO_FS` if it almost be used in `Storage` struct